app/src/split/bluetooth/central_listener.c

```diff
@@ -0,0 +1,73 @@
+#include <logging/log.h>
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+
+#include <zmk/event_manager.h>
+#include <zmk/events/activity_state_changed.h>
+
+static void set_sleep_params(struct bt_conn *conn, void *data) {
+    struct bt_conn_info info;
+
+    bt_conn_get_info(conn, &info);
+
+    if (info.role == BT_CONN_ROLE_MASTER) {
+        int err = bt_conn_le_param_update(conn, BT_LE_CONN_PARAM(0x0012, 0x0012, 10, 400));
```

Comment:
Can we create `#define`s for this and the wake versions to make this a bit clearer?